core/rpc/json/responses.go

```diff
@@ -0,0 +1,73 @@
+package jsonrpc
+
+import (
+	"github.com/kwilteam/kwil-db/core/types"
+	"github.com/kwilteam/kwil-db/core/types/transactions"
+)
+
+// This file defines the response types. There is one for each request type.
+// As with the request types, many are based on types in the core module.
+
+// AccountResponse contains the response object for MethodAccount.
+type AccountResponse struct {
+	Identifier []byte `json:"identifier,omitempty"`
```

Comment:
Potentially.  If we make it `HexBytes` in the requests, then we should make it here.  I'm leaning toward using hex for the account identifiers since in the majority of cases it's an ethereum address, which is hex.  Yeah, will make the change now.

Reply:
cool, I was not sure about this, just think maybe we should.